tests/unit/generator/test_generator.py

```diff
@@ -145,6 +146,50 @@ def test_get_response_ignore_gapic_metadata():
             assert res.file == CodeGeneratorResponse().file
 
 
+@pytest.mark.parametrize("unversioned_package_disabled", (True, False))
+def test_get_response_ignore_unversioned_package(unversioned_package_disabled):
+    g = make_generator()
```

Comment:
Please don't use one character variable names. I see the same construct is used in other tests, but I advocate not perpetuating the practice.

```suggestion
    generator = make_generator()
```

Reply:
Fixed in https://github.com/googleapis/gapic-generator-python/pull/2329/commits/20f40506c9c6596b73312a083ac2765c616c73b7